src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -112,24 +113,15 @@ public synchronized void updateListe(ListeFilme listeEinsortieren, boolean index
                     if (hash.contains(f.getIndex())) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS134)